draft-ietf-scim-device-model.mkd

```diff
@@ -381,9 +381,11 @@ If present, this field contains a dnsName, as specified in Section
 necessarily bind to the incoming IP address of the application client.
 This attribute is not required, read write, singular and NOT case
 sensitive.  This name may also appear as an subjectAlternateName (SAN)
-of type dnsName, in which case the subject MUST be ignored.  If
-multiple dnsNames are present, it is left to server implementations to
-address any authorization conflicts associated with those names.
+of type dnsName, in which case the subject MUST be ignored and the
+domain name MUST match the name of the peer as resolved by a DNS
+reverse lookup.  If a match isn't desired, CAs will not use a dnsName.
+If multiple dnsNames are present, it is left to server implementations
```

Comment:
Can `subjectName` have multiple dnsName values? Or is this talking about the incoming client certificate?

Reply:
No, only the SAN can.

Reply:
Yes, but they are related